### PR TITLE
Fix agile manifesto broken link

### LINKED ISSUE #51: Agile Manifesto link is not working
The agile manifesto link is set as www. however this is not right since it should be a naked url.

The way to fix it is just replacing http://www.agilemanifesto.org by http://agilemanifesto.org

### PR DESCRIPTION
**Pull request by neman**
Fix #51

### CODE REVIEW

**Review by joebew42 — COMMENTED** [review]
Hi Nemanja,

thank you for helping me keeping this study path updated 👍 

I guess we are updating the _wrong_ link, I left a few comments about it.

File: readme.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ Enjoy!
 
 - Get familiar with [the Pomodoro Technique](http://pomodorotechnique.com/) (read the [paper](http://baomee.info/pdf/technique/1.pdf))
 - Read the [Manifesto for Agile Software Development](http://www.agilemanifesto.org/)
```
Comment (joebew42) [inline readme.md]: I guess this was the broken link that needed to be fixed.

File: readme.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ Enjoy!
...
-- Read the [Principles behind the Agile Manifesto](http://www.agilemanifesto.org/principles.html)
+- Read the [Principles behind the Agile Manifesto](http://agilemanifesto.org/)
```
Comment (joebew42) [inline readme.md]: And here we want to keep the link to the _Principles behind the Agile Manifesto_. Consider to remove the `www` part only.

**Review by joebew42 — CHANGES_REQUESTED** [review]

File: readme.md (hunk 2 above)
Comment (neman) [reply readme.md]: Yap, I did it in a hurry, will fix now :/

**Review by joebew42 — APPROVED** [review]
Great work! Thank you Nemanja 👏

**Pull request merged**